### DISCUSSION TITLE
Support for Apache Pulsar

### DISCUSSION THREAD

**Discussion by pjpringlenom** [aklivity/zilla#1223] [Ideas]
Can Zilla support Apache Pulsar as the source?

**Comment by vordimous** [comment]
Not directly, but If you have a Kafka Protocol handler in your Apache Pulsar cluster then Zilla should be able to treat it like any other service that supports the Kafka Protocol. Any of the examples should work the same since Zilla implements the Kafka Protocol spec. 

We haven't directly tested the solutions out there like [starlight-for-kafka](https://github.com/datastax/starlight-for-kafka), [ursa](https://streamnative.io/ursa), etc, but if something isn't working we can investigate the issue and work out if there is some part of the Kafka API that those services don't support.
